nemo/deploy/nlp/query_llm.py

```diff
@@ -170,7 +170,7 @@ def query_llm(
         repetition_penalty: float = None,
         add_BOS: bool = None,
         all_probs: bool = None,
-        compute_logprob: bool = None,
+        log_probs: bool = False,
         end_strings=None,
```

Comment:
Also curious how `all_probs` is different from `log_probs`. It will be good if we can add some docstrings explaining the difference between these two args to avoid confusion.

Reply:
I'll actually remove the all_probs.

Reply:
Removed.